### PR TITLE
Update aklite doc on the conf params

### PR DESCRIPTION
**Pull request by mike-sul**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 10e34359e0f0911620e1196926ae9ef93a2659ee are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1220/docs/artifacts/html/index.html

File: source/reference-manual/ota/aktualizr-lite.rst [hunk 1]
```diff
@@ -105,8 +105,9 @@ The following are aktualizr-repo's configuration parameters that can be useful t
     # A comma separated list of Tags to look for in Targets that should be applied to a given device
     tags = "master"
 
-    # Check if Compose App is installed and running on each update check
-    full_status_check = "0"
+    # The param instructs aktualizr-lite to (re-)create App containers of a new Target just before reboot if set to "1" (default).
+    # If the param is set to "0" then the containers are (re-)created just after a successful boot on a new ostree version during aklite startup.
```
Comment (angolini) [inline source/reference-manual/ota/aktualizr-lite.rst:109]:
```suggestion
    # If the param is set to "0" then the App containers are (re-)created just after a successful boot on a new ostree version during aklite startup.
```

**Review by angolini — APPROVED** [review]
Let me check if I understand it right

The difference between 0 and 1 is if the App container will be create before or after the reboot.

When I read (re-)create I understand that it will create the app container, in case it does not exists already, or re-create it in case it is already there. So, the conclusion for me is that it will always create a App container.

Other than the minor suggestion, LGTM

**Comment by mike-sul** [comment]
> The difference between 0 and 1 is if the App container will be create before or after the reboot.

Correct, (1 - before, 0 - after, default - 1).

**Comment by mike-sul** [comment]
> it will create the app container, in case it does not exists already, or re-create it in case it is already there. So, the conclusion for me is that it will always create a App container.

Correct. On the one hand it's always new container creation, in terms of container IDs, on the other hand it's container re-creation if to look at it from Compose App perspective - replacing of an existing container with a new one.

File: source/reference-manual/ota/aktualizr-lite.rst (hunk 1 above)
Comment (mike-sul) [reply source/reference-manual/ota/aktualizr-lite.rst]: Updated via a forced push.

**Comment by doanac** [comment]
Docs for 1806a0f265ffcfb05fde00cd9bafb4584e7e46e8 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1221/docs/artifacts/html/index.html

**Review by doanac — APPROVED** [review]

**Pull request merged**